bin/validate.js

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env node
+import './.env.js'; // Workaround to ensure `SUPPRESS_NO_CONFIG_WARNING` is set before config is imported
```

Comment:
> Why can't we just assign `process.env` in this file? 😮

Reply:
Is the comment not explicit enough?

Reply:
I mean I understand we need a workaround, I just don't understand why it is needed. I'm fine with that level of documentation in the code, I'm just surprised!

Reply:
It's because all imports are evaluated prior to the body of module doing the importing. So even if we write `process.env.SUPPRESS_NO_CONFIG_WARNING = 'y'` before the `import config`, the instruction is evaluated after, so `config` will still warn.